sdk-extension/opentelemetry-sdk-extension-aws/README.rst

```diff
@@ -74,6 +74,30 @@ populate `resource` attributes by creating a `TraceProvider` using the `AwsEc2Re
 Refer to each detectors' docstring to determine any possible requirements for that
 detector.
 
+
+[WORK IN PROGRESS] Usage (AWS X-Ray Remote Sampler)
+---------------------------------------------------
+
+
+Use the provided AWS X-Ray Remote Sampler by setting this sampler in your instrumented application:
+
+.. code-block:: python
+
+    from opentelemetry.sdk.extension.aws.trace.sampler import AwsXRayRemoteSampler
```

Comment:
```suggestion
    from opentelemetry.sdk.extension.aws.trace.sampler import AwsXRayRemoteSampler
```

Isn't this `_`-prefix though? If you don't want people to use it yet I won't add documentation yet

Reply:
I've undone my changes to the README.